src/tiled/main.cpp

```diff
@@ -278,6 +293,10 @@ int main(int argc, char *argv[])
         return 0;
     }
 
+    QJsonDocument doc(QJsonArray::fromStringList(commandLine.filesToOpen()));
```

Comment:
Whoops! We forgot to precede this with a check like:
```cpp
if (!commandLine.newInstance) {
```
No idea how we missed that. :-)